script.js

```diff
@@ -1,4 +1,332 @@
+  return pushIntoCardArray;
+};
+
+// helper function to sum both cards
+// if jack, queen, king is drawn -> reassign rank to 10
+
+var SumUpCards = function (hand) {
+  var cardInHand1 = hand[0];
+  var cardInHand2 = hand[1];
+  if (
+    cardInHand1.rank == 11 ||
+    cardInHand1.rank == 12 ||
+    cardInHand1.rank == 13
+  ) {
+    cardInHand1.rank = 10;
+  }
+  if (
+    cardInHand2.rank == 11 ||
+    cardInHand2.rank == 12 ||
+    cardInHand2.rank == 13
+  ) {
+    cardInHand2.rank = 10;
+  }
+  var sumValue = cardInHand1.rank + cardInHand2.rank;
+  return sumValue;
+};
+
+var SumUpCardsSecondRound = function (hand) {
+  var cardInHand3 = hand[2];
+  if (
+    cardInHand3.rank == 11 ||
+    cardInHand3.rank == 12 ||
+    cardInHand3.rank == 13
+  ) {
+    cardInHand3.rank = 10;
+  }
+  var newValue = cardInHand3.rank;
+  return newValue;
+};
+
+var givePlayerOutputValue = function () {
+  return (DefaultOutputValue =
+    "PLAYER: " +
+    playerHand[0].name +
+    " of " +
+    playerHand[0].suit +
+    " and " +
+    playerHand[1].name +
+    " of " +
+    playerHand[1].suit +
+    ".");
+  // "<br><br>COMPUTER: " +
+  // computerHand[0].name +
+  // " of " +
+  // computerHand[0].suit +
+  // "<br>"
+};
+
+var giveComputerOutputValue = function () {
+  return (DefaultOutputValue =
+    "<br><br>COMPUTER: " +
+    computerHand[0].name +
+    " of " +
+    computerHand[0].suit +
+    ".");
+};
+
+var giveThirdIteminOutputValue = function () {
+  return (ThirdItemOutputValue =
+    "<br><br> Your 3rd card is " +
+    playerHand[2].name +
+    " of " +
+    playerHand[2].suit +
+    ".");
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (playerHand.length == 0) {
+    // drawing first and second card for player and computer
+    dealCard(playerHand);
+    dealCard(computerHand);
+    dealCard(playerHand);
+    dealCard(computerHand);
+
+    playerHandTotal = SumUpCards(playerHand);
+    computerHandTotal = SumUpCards(computerHand);
+
+    console.log("player card sum: " + playerHandTotal);
+
+    return (
+      givePlayerOutputValue() +
+      giveComputerOutputValue() +
+      "<br><br> Your total value is " +
+      playerHandTotal +
+      ". Submit 'hit' or 'stand'."
+    );
+  }
+
+  // checking for blackjack
+  if (playerHand.length == 2 && playerHandTotal == 21) {
+    console.log("player card sum: " + playerHandTotal);
+    return (
+      givePlayerOutputValue() +
+      giveComputerOutputValue() +
+      "Your total value is " +
+      playerHandTotal +
+      ". Player wins blackjack! Refresh the page to play again."
+    );
+  } else if (playerHand.length == 2 && computerHandTotal == 21) {
+    console.log("computer card sum: " + computerHandTotal);
+    return (
+      givePlayerOutputValue() +
+      giveComputerOutputValue() +
+      "Your total value is " +
+      computerHandTotal +
+      ". Computer wins blackjack! Refresh the page to play again."
+    );
+  }
```

Comment:
since we repeated  this logic, we could make it into a function and reuse it for both player and computer